examples/tests/linuxbuild.py

```diff
@@ -24,7 +24,8 @@ def setUp(self):
 
         self.linux_build = kernel.KernelBuild(kernel_version,
                                               linux_config,
-                                              self.srcdir)
+                                              self.srcdir,
+                                              ['/var/tmp'])
```

Comment:
Well I had something like this: https://github.com/ldoktor/avocado/commits/1248 in mind, what do you think about that?